cmd/dagger/module.go

```diff
@@ -368,7 +405,16 @@ This command is idempotent: you can run it at any time, any number of times. It
 			}
 			// if SDK is set but source path isn't and the user didn't provide --source, we'll use the default source path
 			if modSDK != "" && modSourcePath == "" && developSourcePath == "" {
-				developSourcePath = filepath.Join(modConf.LocalRootSourcePath, defaultModuleSourceDirName)
+				inferredSourcePath, err := inferSourcePathDir(modConf.LocalRootSourcePath)
+				if err != nil {
+					return err
+				}
+
+				if inferredSourcePath != "" {
+					developSourcePath = filepath.Join(modConf.LocalRootSourcePath, inferredSourcePath)
+				} else {
+					developSourcePath = filepath.Join(modConf.LocalRootSourcePath, defaultModuleSourceDirName)
```

Comment:
This `const` is back? https://github.com/dagger/dagger/pull/8557#discussion_r1808451125

Reply:
yep, I made the changes from scratch and missed it. thanks. its removed now.